src/ansys/mapdl/core/_commands/post1/_fatigue.py

```diff
@@ -0,0 +1,652 @@
+
+        """
+        command = f"FELIST,{nev1},{nev2},{ninc}"
+        return self.run(command, **kwargs)
+
+    def fl(
+        self,
+        nloc: str = "",
+        node: str = "",
+        scfx: str = "",
+        scfy: str = "",
+        scfz: str = "",
+        title: str = "",
+        **kwargs,
+    ):
+        r"""Defines a set of fatigue location parameters.
+
+        Mechanical APDL Command: `FL <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_FL.html>`_
+
+        Parameters
+        ----------
+        nloc : str
+            Reference number for this location (within ``MXLOC`` ). When defining a new location, defaults
+            to lowest unused location. If the specified ``NODE`` is already associated with a location,
+            ``NLOC`` defaults to that existing location.
+
+        node : str
+            Node number corresponding to this location (must be unique). Used only to associate a node with
+            a new location or to find an existing location (if ``NLOC`` is not input). If ``NODE`` = -1 (or
+            redefined), erase all parameters and fatigue stresses for this location.
+
+        scfx : str
+            Stress concentration factors applied to the total stresses. Factors are applied in the global X,
+            Y, and Z directions unless the axisymmetric option of the :ref:`fssect` is used (that is,
+            ``RHO`` is nonzero), in which case the factors are applied in the section x, y, and z (radial,
+            axial, and hoop) directions.
+
+        scfy : str
+            Stress concentration factors applied to the total stresses. Factors are applied in the global X,
+            Y, and Z directions unless the axisymmetric option of the :ref:`fssect` is used (that is,
+            ``RHO`` is nonzero), in which case the factors are applied in the section x, y, and z (radial,
+            axial, and hoop) directions.
+
+        scfz : str
+            Stress concentration factors applied to the total stresses. Factors are applied in the global X,
+            Y, and Z directions unless the axisymmetric option of the :ref:`fssect` is used (that is,
+            ``RHO`` is nonzero), in which case the factors are applied in the section x, y, and z (radial,
+            axial, and hoop) directions.
+
+        title : str
+            User-defined title for this location (up to 20 characters).
+
+        Notes
+        -----
+
+        .. _FL_notes:
+
+        Repeat FL command to define additional sets of location parameters ( ``MXLOC`` limit), to redefine
+        location parameters, or to delete location stress conditions.
+
+        One location must be defined for each node of interest and only one node can be associated with each
+        location. See the FTSIZE command for the maximum locations ( ``MXLOC`` ) allowed. A location will be
+        automatically defined for a node not having a location when the FSSECT, FSNODE, or FS command is
```

Comment:
I guess these commands are not highlighted as such in the MAPDL documentation.

Reply:
I saw a warning around...  for instance: https://cautious-adventure-g6vqq8v.pages.github.io/version/dev/post1/_autosummary/ansys.mapdl.core._commands.post1._fatigue.Fatigue.fl.html#ansys.mapdl.core._commands.post1._fatigue.Fatigue.fl

Reply:
I'm not sure which other warning you mean...

Reply:
Ow that's good then, that's what I meant! I forgot I implemented it.